src/revolve/nodes/report.py

```
@@ -0,0 +1,81 @@
+        message="All done",
+        description="All done"
+    )
+    
+    db_name = os.environ.get("DB_NAME")
+    db_user = os.environ.get("DB_USER")
+    db_password = os.environ.get("DB_PASSWORD")
+    db_host = os.environ.get("DB_HOST")
+    db_port = os.environ.get("DB_PORT")
+    static_dir = os.environ.get("STATIC_DIR", "-")
+
+    env_file = open(f"{get_source_folder()}/.env", "w")
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)